src/main/java/com/adyen/model/checkout/PaymentMethodsRequest.java

```diff
@@ -81,6 +81,10 @@
     @SerializedName("trustedShopper")
     private Boolean trustedShopper = null;
 
+    public PaymentMethodsRequest() {
+        applicationInfo = new ApplicationInfo();
+    }
+
```

Comment:
This is not needed here. ApplicationInfo is used only in Payment submission.

Reply:
Reverted.